app/models/service_template_ansible_playbook.rb

```diff
@@ -151,4 +151,17 @@ def update_catalog_item(options, auth_user = nil)
     end
     super
   end
+
+  def destroy(auth_user = 'system')
```

Comment:
`destroy` and `destroy!` are AR methods. Should we override it/them this way? Can we add an argument?. Let's do some research on a Rails way to clean resources at destroying.